Sources/PayPalNativePayments/NativeCheckoutStartable.swift

```diff
@@ -6,16 +6,26 @@ import CorePayments
 #endif
 
 protocol NativeCheckoutStartable {
+    
+    typealias StartabeApproveCallback = (String, String) -> Void
+    typealias StartabeShippingCallback = (
+        ShippingChangeType,
+        PayPalNativePaysheetActions,
+        PayPalNativeShippingAddress,
+        PayPalNativeShippingMethod?
+    ) -> Void
+    typealias StartabeCancelCallback = () -> Void
+    typealias StartabeErrorCallback = (String) -> Void
 
-    // swiftlint:disable function_parameter_count
+
+    // swiftlint:disable:next function_parameter_count
     func start(
         presentingViewController: UIViewController?,
```

Comment:
how come we need to explicitly pass in presentingVC? I know in Braintree, we don't pass this param and I think it defaults to root most vc?

Reply:
i think in BT it was done for consistency reasons maybe @scannillo? Here if the merchant doesnt pass the VC it will default to root.